plugins/modules/mongodb_replicaset.py

```diff
@@ -554,8 +554,13 @@ def main():
     debug = module.params['debug']
     cluster_cmd = module.params['cluster_cmd']
 
+    voting_members = []
+    for member in members:
+        if ("votes" in member and member["votes"]) or "votes" not in member:
```

Comment:
If I remember correctly "votes" is an integer and technically this should be summed. Let's leave that for another "fixer"!